lib/charms/mongodb/v1/shards_interface.py

```diff
@@ -253,12 +243,6 @@ def remove_shards(self, departed_shard_id):
                 self.charm.unit.status = MaintenanceStatus(f"Draining shard {shard}")
                 logger.info("Attempting to removing shard: %s", shard)
                 mongo.remove_shard(shard)
-                logger.info("Shard: %s, is now draining", shard)
```

Comment:
we can remove this since `remove_shard` only succeeds on complete & successful removal/drainage